app/components/footer/talk_to_us_component.html.erb

```diff
@@ -24,7 +24,7 @@
             <p>
               An experienced teaching professional can help if you're ready to get into
               teaching or if you're returning to teaching and qualified to teach maths,
```

Comment:
I wonder if we need to reword this - as I understand it, it's not really 'qualified to teach maths' etc as QTS is not subject-specific - it's more about want to teach maths etc (and have QTS).

However, this is going to make it very long-winded if we add the QTS bit in specifically - any ideas?